### PR TITLE
[BACKLOG-11789]-Text input fields overflow parent in DB Connection dialog in IE 11

### PR DESCRIPTION
**Pull request by eliofreitas**

### CODE REVIEW

File: pentaho-gwt-widgets/src/org/pentaho/gwt/widgets/themes/public/themes/sapphire/globalSapphire.css [hunk 1]
```diff
@@ -4411,6 +4411,10 @@ span.gwt-RadioButton label {
   margin-top: -3px;
 }
 
+#contentCleanerPanel tbody tr td table tbody tr td table tbody tr td .gwt-TextBox {
```
Comment (afrjorge) [inline pentaho-gwt-widgets/src/org/pentaho/gwt/widgets/themes/public/themes/sapphire/globalSapphire.css]: Do we need all these HTML elements here?

Comment (annapopova1) [inline pentaho-gwt-widgets/src/org/pentaho/gwt/widgets/themes/public/themes/sapphire/globalSapphire.css]: Can't understand this limitation 50px.. Looks like box-sizing: border-box; fixes the issue.

File: pentaho-gwt-widgets/src/org/pentaho/gwt/widgets/themes/public/themes/sapphire/globalSapphire.css [hunk 2]
```diff
@@ -4411,6 +4411,10 @@ span.gwt-RadioButton label {
   margin-top: -3px;
 }
 
+#contentCleanerPanel .gwt-TextBox {
```
Comment (annapopova1) [inline pentaho-gwt-widgets/src/org/pentaho/gwt/widgets/themes/public/themes/sapphire/globalSapphire.css]: IMO it's strange to have specific style for concrete #contentCleanerPanel here in commons gwt modules project.
This element with id "contentCleanerPanel" is created in PUC where also located css styles for different themes. Does it make sense to move this specific style to user-console project?

**Comment by eliofreitas** [comment]
https://github.com/pentaho/pentaho-platform/pull/3196 merge with this one

**Pull request merged**